src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -434,6 +441,18 @@ public function getFName()
         return $this->fname;
     }
 
+     /**
+     * Returns the User's first name.
```

Comment:
this does not return the first name

Reply:
yeah this a copy paste whoops forget to change it